airflow/providers/daskexecutor/CHANGELOG.rst

```diff
@@ -27,6 +27,12 @@
 Changelog
 ---------
 
+1.1.1
+.....
+
+* Mark daskececutor provider as removed
```

Comment:
Lets add link to the mailing list discussion?

Reply:
Added